qds_sdk/cluster.py

```diff
@@ -353,6 +353,28 @@ def update(cls, cluster_id_label, cluster_info):
         return conn.put(cls.element_path(cluster_id_label), data=cluster_info)
 
     @classmethod
+    def _parse_cluster_manage_command(cls, args):
+        """
+        Parse command line arguments for cluster manage commands.
+        """
+        argparser = ArgumentParser(prog="cluster_manage_command")
+        group = argparser.add_mutually_exclusive_group(required=True)
+        group.add_argument("--id", dest="cluster_id",
+                          help="execute on cluster with this id")
+        group.add_argument("--label", dest="label",
```

Comment:
Remember to add Required=True. Refer to https://docs.python.org/2/library/argparse.html